.gitignore

```
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# .DS_Store files
+**/.DS_Store
```

```suggestion
.DS_Store
```
otherwise it wouldn't handle top level